docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -34,7 +34,8 @@ This section guides you through the Semgrep Pro Engine installation and helps yo
 It is **recommended** to run Semgrep Pro Engine with Semgrep Cloud Platform (SCP). This enables you to view and triage your findings from a centralized location. Code is not uploaded.
```

Comment:
Not related to this update, but we might want to review this "Code is not uploaded" sentence at some point :)

Reply:
We had a small discussion about this during the Semgrep Assistant beta. We can keep the wording for the product as overall we don't upload code with 2 exceptions, and they are clearly labeled as exceptions:
- Code access (Zero config scanning)
- Semgrep Assistant

Does this make sense?

Reply:
sure!